README.md

```diff
@@ -6,6 +6,8 @@
 
 We welcome your contributions to our website. If you have a proposal you want to share with us, simply keep the following things in mind.
 
+> You only need to follow the installation instructions below if you want to contribute something.
```

Comment:
I suggest you put the following header before this line:

`## Contributing`

This would make it very clear all the other sections belong to a contributing area.

Reply:
You're right, thanks. I fixed it.